instrumentation/rack/lib/opentelemetry/instrumentation/rack/instrumentation.rb

```diff
@@ -29,12 +28,56 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         option :url_quantization,         default: nil,   validate: :callable
         option :untraced_requests,        default: nil,   validate: :callable
         option :response_propagators,     default: [],    validate: :array
+        # This option is only valid for applicaitons using Rack 2.0 or greater
+        option :use_rack_events,          default: false, validate: :boolean
+
+        # Temporary Helper for Sinatra and ActionPack middleware to use during installation
```

Comment:
What's the long term plan (you may have answered this elsewhere)?

Reply:
This configuration setting and helper will exist until...

1. We drop support for Sinatra versions that require Rack 1.x
2. A future release that makes the Event Handler the default way for users to instrument their applications